pkg/utils/report/results.go

```diff
@@ -136,6 +136,14 @@ func ToPolicyReportResult(pol engineapi.GenericPolicy, ruleResult engineapi.Rule
 		kyvernoPolicy := pol.AsKyvernoPolicy()
 		result.Source = SourceKyverno
 		process = selectProcess(kyvernoPolicy.BackgroundProcessingEnabled(), kyvernoPolicy.AdmissionProcessingEnabled())
+	case pol.AsMutatingAdmissionPolicy() != nil:
+		result.Source = SourceMutatingAdmissionPolicy
+		result.Policy = ruleResult.Name()
+		process = "admission review"
+		if binding := ruleResult.MutatingAdmissionPolicyBinding(); binding != nil {
+			addProperty("binding", binding.Name, &result)
+		}
```

Comment:
Can you remove this change? It is related to generating reports for MAPs, and we don't have it now.